src/controllers/requestController.js

```diff
@@ -0,0 +1,34 @@
+import { RequestService } from '../services';
+import { Helpers } from '../utils';
+
+
+const { getRequests } = RequestService;
+const { successResponse, errorResponse } = Helpers;
+
+/**
+ * A collection of methods that controls user requests.
+ *
+ * @class RequestController
+ */
+export default class RequestController {
+  /**
+ *  assign a role to a user
+ * @static
+ * @param {Request} req - The request from the endpoint.
+ * @param {Response} res - The response returned by the method.
+ * @returns { JSON } - A JSON object containing success or failure details.
+ * @memberof RequestController
+ */
+  static async getUserRequests(req, res) {
+    try {
+      const { id } = req.data;
+      const requests = await getRequests(id);
+      if (!requests[0]) {
```

Comment:
Wouldn't checking the length of the request array be a better approach?

Reply:
I wasn't aware there was a better one of this approach or the array length approach, I just went for the shorter one. But since you're suggesting it now I would update this to check the length of the array.  Thanks